JS-Algorithms/challenges.js

```diff
@@ -118,7 +155,16 @@ Because the 12th index in the Fibonacci sequence is 144, and 144 has three digit
 ***** */
 
 const fibIndex = (n) => {
-  // YOUR CODE HERE...
+  let fib = n,
+    n2 = 0,
+    n1 = 1;
```

Comment:
for readibility, just keep the declaration on this way:

let fib = n
let n2 = 0
let  n1 = 1

A more readable code is easier to understand and maintain. Just keep clear and simple